terraform/variables.tf

```diff
@@ -71,3 +71,15 @@ variable "sgtm_feature__followup_review_github_users" {
   description = "A comma-separated list of Github usernames that require follow-up review after merge"
   default     = ""
 }
+
+variable "sgtm_feature__check_rerun_threshold_hours" {
+  type = number
+  description = "Number of hours after which a check run should be rerequested. Must be combined with sgtm_feature__check_rerun_base_ref_names."
+  default = 0
+}
+
+variable "sgtm_feature__check_rerun_base_ref_names" {
+  type        = string
+  description = "A comma-separated list of base refs that will trigger check run rerequests when stale. Must be combined with sgtm_feature__check_rerun_freshness_duration_hours."
+  default     = ""
```

Comment:
Do you think the default here should be `"main,master"`?

Reply:
I think I'll leave this disabled by default. Rerunning check runs is probably not expected behaviour for most workflows (and may incur unexpected costs)

Reply:
I was thinking more that most orgs only open PRs against `main` or `master`, so defaulting that without defaulting the staleness window still doesn't enable the feature, but if you set a `SGTM_FEATURE__CHECK_RERUN_THRESHOLD_HOURS`, that would enable the feature without having to _also_ set a branch name unless you had a unique flow (like ours). Just a thought though, either way works!